### PR TITLE
Update UG and DG

### PR DESCRIPTION
**Pull request by donwong2308**
update timestamp format and case insensitive for day format

### CODE REVIEW

**Review by JustWeiHao — APPROVED** [review]
LGTM

**Review by PallonCX — APPROVED** [review]
LGTM

**Review by PallonCX — APPROVED** [review]
LGTM

File: docs/UserGuide.md [hunk 1]
```diff
@@ -241,9 +242,10 @@ List all contacts and their details.
 ### Checking who is free : `whoisfree`
 View all persons that are available on the specified day and time.
 
-Format: `whoisfree DAY:TIME`
-- `DAY` is from Mon-Sun.
-- `TIME` is 24-hour time format.
+Format: `whoisfree TIMESTAMP`
```
Comment (nyankoclaws) [inline docs/UserGuide.md:245]: I feel without the example it's a little unclear.. but w the example is ok ish??

**Review by nyankoclaws — APPROVED** [review]
Minor nitpick, LGTM otherwise

**Review by larainezo — APPROVED** [review]
LGTM

**Pull request merged**